docs/config/index.rst

```diff
@@ -31,15 +31,13 @@ can find the exact location by doing::
     >>> get_config_dir()
 
 And you should see the location of your configuration directory. The standard
-scheme generally puts your configuration directory in ``$HOME/.astropy/config``,
-but if you've set the environment variable ``XDG_CONFIG_HOME`` and the
-``$XDG_CONFIG_HOME/astropy`` directory exists, it will instead be there.
-
-.. note::
-    This is a slight variation from the behavior of most Linux
-    applications with respect to ``$XDG_CONFIG_HOME``, where the
-    default, if ``$XDG_CONFIG_HOME`` is not defined, would be to put
-    configuration in ``~/.astropy/config``.
+scheme generally puts your configuration directory in
+``$HOME/.astropy/config``. It can be customized with the environment variable
+``XDG_CONFIG_HOME`` and the ``$XDG_CONFIG_HOME/astropy`` directory must exists.
```

Comment:
I thought "must exist" is more grammatically correct but I am not 100% sure.

Reply:
Ok thanks, I will update this and squash.